tests/cupyx_tests/scipy_tests/sparse_tests/test_linalg.py

```diff
@@ -1268,3 +1268,28 @@ def test_spilu_0(self, dtype, xp, sp):
         else:
             ainv = sp.linalg.spilu(a)
         return ainv.solve(b)
+
+
+@testing.parameterize(*testing.product({
+    'damp': [0, 1, 2]
+}))
+@testing.gpu
```

Comment:
Now we don't need to use `@testing.gpu` as it was originally introduced when CuPy was a part of Chainer, and does not make any use in CuPy. (Recent [change](https://github.com/cupy/cupy/wiki/Development-policies/_compare/caddfd6f010b3c93a90dd8820db9c0b7426cd86e...072ee9bf3da3d15119f4620b67cbb5aae03eaa18) in our development policy )